### PR TITLE
Daemon: Bug in env export

### PR DESCRIPTION
**Pull request by ppp0**
While debuggin Elasticsearch, I found that the environment in Wheezy/init is not properly exported

### CODE REVIEW

**Comment by ppp0** [comment]
@kris-lab please review

File: modules/daemon/templates/sysvinit.sh.erb [hunk 1]
```diff
@@ -16,7 +16,9 @@ DAEMON_ARGS='<%= @args %>'
 PIDFILE='/var/run/<%= @name %>.pid'
 USER='<%= @user %>'
 <% unless @env.empty? %>
-export<% @env.each do |name, value| -%> <%= name %>='<%= value %>'<% end -%>
+<% @env.each do |name, value| %>
+export <%= name %>='<%= value %>'
+<% end %>
```
Comment (kris-lab) [inline modules/daemon/templates/sysvinit.sh.erb]: Added comment at https://github.com/cargomedia/puppet-packages/pull/1481#discussion-diff-77520211

**Comment by kris-lab** [comment]
Small comment in the code. The rest looks good.

**Comment by kris-lab** [comment]
lgtm

**Comment by ppp0** [comment]
After discussing and testing we found out this change not to be needed

**Pull request closed without merging**